docs/source/python/extensions.rst

```diff
@@ -92,17 +92,17 @@ In addition to extension classes, there are a few helper functions to ease handl
 
    A dictionary mapping of a unique key to an extension class::
 
-      >>> KEY_TO_EXTENSION['authority_information_access']
-      <class 'django_ca.extensions.extensions.AuthorityInformationAccess'>
+      >>> KEY_TO_EXTENSION['authority_information_access'].__name__
+      'AuthorityInformationAccess'
```

Comment:
There was an anomaly on the outcome between Python 3.6 vs. Python >=3.7.

I don't actually know why, but on Python 3.6, it said
```python
'django_ca.extensions.extensions.AuthorityInformationAccess'
```
instead of
```python
<class 'django_ca.extensions.extensions.AuthorityInformationAccess'>
```

With this slight amendment, it works on all designated versions of Python 3.x.

Reply:
This adjustment has been reverted.